dist/ExtUtils-ParseXS/lib/ExtUtils/ParseXS/Utilities.pm

```diff
@@ -327,6 +327,54 @@ sub map_type {
   return $type;
 }
 
+=head2 C<standard_XS_version_cmp()>
+
+=over 4
+
+=item * Purpose
+
+Provides the definition for PERL_VERSION_* compare macros.
+
+=item * Arguments
+
+None.
+
+=item * Return Value
+
+Returns a string to define PERL_VERSION.
+
+=back
+
+=cut
+
+sub standard_XS_version_cmp {
+    # these macros are now part of the API since 5.033
+    if ( $] > 5.033 ) {
```

Comment:
`$] >= 5.033`?

Reply:
using `$] >= 5.033` is incorrect, a better check would be `$] >= 5.033001` as `5.033000` does not provide the compare macro

Reply:
changed the logic as part of e7bce4d650f714e804b41f93c57f8ee243e8b049 to be more clear